### ISSUE TITLE
domainguess, regex

### ISSUE THREAD

**Issue by runout-at** [vexim/vexim2#135]
Would be nice to have a config variable to strip subdomains for domainguess like:
```php
$domainguess_lefttrim = "mail|vexim";
```
for usage in index.php (line 20) 
```php
if($domainguess===1) echo '@'.preg_replace ("/^".$domainguess_lefttrim."\./", "", $_SERVER["SERVER_NAME"]);
```
and login.php (line 14)
```php
if($domainguess === 1 && $_POST['username']!=='siteadmin') $_POST['username'].='@'.preg_replace ("/^".$domainguess_lefttrim."\./", "", $_SERVER["SERVER_NAME"]);
```

**Comment by Udera** [comment]
Is this tested? From my POV, we could include this if you create a pull request. @rimas-kudelis what do you think?

**Comment by rimas-kudelis** [comment]
Fixed by #151
